### ISSUE TITLE
which version of tensorflow is required by byteps

### ISSUE THREAD

**Issue by wangshuaizs** [bytedance/byteps#25]
I installed tensorflow 1.10, but when I ran byteps, the logs showed ```AttributeError: 'module' object has no attribute 'v1'```. I think it might be caused by my mismatched tf. So, which version of tensorflow does byteps support?

```
$ DMLC_ROLE=worker DMLC_PS_ROOT_URI=12.12.10.12 DMLC_PS_ROOT_PORT=9000 DMLC_WORKER_ID=0 DMLC_NUM_WORKER=1 DMLC_NUM_SERVER=1 python launcher/launch.py python example/tensorflow/tensorflow_mnist.py
BytePS launching worker
INFO:tensorflow:Create CheckpointSaverHook.
Traceback (most recent call last):
  File "example/tensorflow/tensorflow_mnist.py", line 160, in <module>
    tf.app.run()
  File "/home/shuai/.conda/envs/byteps/lib/python2.7/site-packages/tensorflow/python/platform/app.py", line 125, in run
    _sys.exit(main(argv))
  File "example/tensorflow/tensorflow_mnist.py", line 152, in main
    config=config) as mon_sess:
  File "/home/shuai/.conda/envs/byteps/lib/python2.7/site-packages/tensorflow/python/training/monitored_session.py", line 421, in MonitoredTrainingSession
    stop_grace_period_secs=stop_grace_period_secs)
  File "/home/shuai/.conda/envs/byteps/lib/python2.7/site-packages/tensorflow/python/training/monitored_session.py", line 832, in __init__
    stop_grace_period_secs=stop_grace_period_secs)
  File "/home/shuai/.conda/envs/byteps/lib/python2.7/site-packages/tensorflow/python/training/monitored_session.py", line 548, in __init__
    h.begin()
  File "/home/shuai/.conda/envs/byteps/lib/python2.7/site-packages/byteps-0.1.0-py2.7-linux-x86_64.egg/byteps/tensorflow/__init__.py", line 107, in begin
    self.bcast_op = broadcast_global_variables(self.root_rank)
  File "/home/shuai/.conda/envs/byteps/lib/python2.7/site-packages/byteps-0.1.0-py2.7-linux-x86_64.egg/byteps/tensorflow/__init__.py", line 66, in broadcast_global_variables
    return broadcast_variables(tf.global_variables(), root_rank, scope)
  File "/home/shuai/.conda/envs/byteps/lib/python2.7/site-packages/byteps-0.1.0-py2.7-linux-x86_64.egg/byteps/tensorflow/__init__.py", line 78, in broadcast_variables
    for var in variables])
  File "/home/shuai/.conda/envs/byteps/lib/python2.7/site-packages/byteps-0.1.0-py2.7-linux-x86_64.egg/byteps/tensorflow/ops.py", line 116, in broadcast
    scope = tf.compat.v1.get_default_graph().get_name_scope()
AttributeError: 'module' object has no attribute 'v1'
^C^C^C^C^C^C^C^C^C^CTerminated
```

**Comment by ymjiang** [comment]
Can you please update to the latest commit? We just fix this problem: d29964329494f790776e09197629ad823391cef5

**Comment by wangshuaizs** [comment]
@ymjiang , Thanks, it worked!
